recipes/yapss/meta.yaml

```
@@ -0,0 +1,144 @@
+{% set name = "yapss" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "ef084862f31dddd2c16e53ff60014768a64fb9e7266d75af5477dd3f1f44e405" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/y/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<3.14
```

Please have a look at the linter
```suggestion
    - python {{ python_min }}
```

---

Done, on all instances.